src/pages/ScholarshipForm.jsx

```diff
@@ -1,115 +1,134 @@
+            label="Scholarship Name *"
+            error={formik.touched.name && Boolean(formik.errors.name)}
+            helperText={(formik.touched.name && formik.errors.name) || ' '}
+            value={formik.values.name}
+            onChange={formik.handleChange}
             fullWidth
           />
         </div>
+
         <DatePicker
+          className={classes.fieldStye}
           id="deadline"
-          label="Deadline"
-          error={errors.deadline}
-          helperText={(errors.deadline && 'Please enter a valid date') || ' '}
-          value={values.deadline}
-          onChange={(date) => setValues({ ...values, deadline: date })}
+          label="Deadline *"
+          error={formik.touched.deadline && Boolean(formik.errors.deadline)}
+          helperText={
+            (formik.touched.deadline && formik.errors.deadline) || ' '
+          }
+          value={formik.values.deadline}
+          onChange={(date) => formik.setFieldValue('deadline', date, false)}
         />
+
         <div>
           <TextField
+            className={classes.fieldStye}
             id="description"
-            label="Description"
-            error={errors.description}
-            helperText={errors.description ? 'Please enter a description' : ' '}
-            required
-            value={values.description}
-            onChange={(e) =>
-              setValues({ ...values, description: e.target.value })
+            label="Description *"
+            error={
+              formik.touched.description && Boolean(formik.errors.description)
+            }
+            helperText={
+              (formik.touched.description && formik.errors.description) || ' '
             }
+            value={formik.values.description}
+            onChange={formik.handleChange}
+            multiline
             fullWidth
           />
         </div>
+
         <div>
           <TextField
+            className={classes.fieldStye}
             id="website"
-            label="Website"
-            error={errors.website}
-            helperText={errors.website ? 'Please enter a valid website' : ' '}
-            required
-            value={values.website}
-            onChange={(e) => setValues({ ...values, website: e.target.value })}
+            label="Website *"
+            error={formik.touched.website && Boolean(formik.errors.website)}
+            helperText={
+              (formik.touched.website && formik.errors.website) || ' '
+            }
+            value={formik.values.website}
+            onChange={formik.handleChange}
             fullWidth
           />
         </div>
+
         <ScholarshipAmountField
-          helperText={errors.amountHelperText}
-          amountType={values.amountType}
-          minAmount={values.minAmount}
-          maxAmount={values.maxAmount}
+          helperText={
+            (getIn(formik.touched, 'amount.type') &&
+              getIn(formik.errors, 'amount.type')) ||
```

Comment:
I'm not familiar with formik but maybe `formik.touched.amount.type`, etc. works?

Reply:
Or if not, I imagine `formik.touched['amount.type']` probably does.

Reply:
The first suggestion throws an error while the 2nd suggestion doesn't access the values.

Reply:
Ah alright